### PR TITLE
Integration landing page

### LINKED ISSUE #80: IN: Landing Page

### PR DESCRIPTION
**Pull request by samsoncsyu7777**
### What this PR does (required):
- create FindSitterForm interface
- create FindSitterFormContext to save city, startDate and endDate for profile listing
- route to dashboard to see the profile listing after saving the details in context

### Screenshots / Videos (front-end only):

Screenshot 2021-11-27 at 7 54 36 AM

### Any information needed to test this feature (required):
- http://localhost:3000/

### CODE REVIEW

**Review by sundayezeilo — COMMENTED** [review]
Good work with this one :+1:

File: client/src/context/useFindSitterFormContext.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { FindSitterForm } from '../interface/FindSitterForm';
+import { createContext, FunctionComponent, useState, useContext, useCallback } from 'react';
+
+interface IFindSitterFormContext {
```
Comment (rajivtitus) [inline client/src/context/useFindSitterFormContext.tsx:4]: You don't need to prefix interfaces with 'I' in JS :)

File: client/src/context/useFindSitterFormContext.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  findSitterFormContext: FindSitterForm | null | undefined;
+  updateFindSitterFormContext: (findSitterForm: FindSitterForm) => void;
+}
+
+export const FindSitterFormContext = createContext<IFindSitterFormContext>({
+  findSitterFormContext: undefined,
+  updateFindSitterFormContext: () => null,
+});
+
+export const FindSitterFormProvider: FunctionComponent = ({ children }): JSX.Element => {
```
Comment (rajivtitus) [inline client/src/context/useFindSitterFormContext.tsx:14]: Good thinking but we don't need a separate context, you can include the `findSitterFormContext` state within the SittersProvider (the context which contains sitter profiles) itself 😄

**Review by rajivtitus — COMMENTED** [review]
Good work here 👍🏽

**Pull request merged**